test/test.googleauth.ts

```diff
@@ -1295,3 +1295,14 @@ it('should get the current environment if GAE', async () => {
   const env = await auth.getEnv();
   assert.equal(env, envDetect.GCPEnv.APP_ENGINE);
 });
+
+it('should make the request', async () => {
+  const url = 'http://example.com';
+  const {auth, scopes} = mockGCE();
+  const data = {breakfast: 'coffee'};
```

Comment:
just coffee? no eggs? :)

Reply:
🤷‍♂️  I worked from home this morning, too lazy to make my own.